experiments/calibration/run_calibration.jl

```diff
@@ -11,22 +11,39 @@ import JLD2
 include(joinpath(pkgdir(ClimaLand), "experiments/calibration/api.jl"))
 
 const CALIBRATE_CONFIG = CalibrateConfig(;
-    short_names = ["lwu"],
+    short_names = ["swu"],
     minibatch_size = 1,
-    n_iterations = 3,
-    sample_date_ranges = [("2007-12-1", "2008-9-1")],
+    n_iterations = 6,
+    sample_date_ranges = [("2008-12-1", "2009-9-1"), ("2009-12-1", "2010-9-1"), ("2010-12-1", "2011-9-1"), ("2011-12-1", "2012-9-1"), ("2012-12-1", "2013-9-1"), ("2013-12-1", "2014-9-1")],
```

Comment:
Im a little confused why these range from Dec 1 to Sept 1. This seems tied to the spinup time. Could you leave a comment?